README.md

```diff
@@ -111,18 +133,12 @@ public class MyTest : MyTestBase
 {
 }
 ```
-2. Attach the [assembly resolve event handler](https://docs.microsoft.com/en-us/dotnet/standard/assembly/resolve-loads) to [MSBuildAssemblyResolver.AssemblyResolve](https://github.com/jeffkl/MSBuildProjectCreator/blob/main/src/MSBuildProjectCreator/MSBuildAssemblyResolver.cs)
```

Comment:
Let's leave this option for people as a last resort or we can tell them to call `MSBuildAssemblyResolver.Register()` in a static constructor of their test or app?
```
public static MyUnitTests
{
    MSBuildAssemblyResolver.Register();
}
```

Reply:
I didn't like the instructions to add the event handler, since it seems to be incomplete (doesn't set environment variables). I'm happy to clarify though that they can call it in their own static constructor.

Since the module initializer is conceptually similar, I'll see if I can refactor the docs slightly to make it clearer. Thanks!

Reply:
Updated! Let me know if this reads a bit cleaner for you!